### DISCUSSION TITLE
Get() doesn't seems to find object created by addLevel()

### DISCUSSION THREAD

**Discussion by martiansparlor** [replit/kaboom#857] [Q&A] [answered]
Hi,
In the v2000 I used to count or find objects on the scene in the a load() function, using get().
This way I could for example use a symbol with a "start" label in addLevel() to place a different statring position for the player in each level.
With the v3000, get() seems to be only able to find object declared in the scene scope but not the one created via addLevel(). Is there a way around it ? I'm looking for a simple solution that I can use with childs during classes.
Thanks for the amazing work by the way !

**Comment by lajbel** [comment, marked as answer]
In Kaboom 3000, you can utilize `get("*", { recursive: true })` to access to the scene root's game objects along with its child objs. This is due to the parent/child system where a level serves as the parent and the various tiles represent the children.

You can also use `obj.get()` to retrieve children within a parent object. EX:
```js
const level = addLevel(...);

level.get("*"); // Get all children of level
```

It's wonderful to hear that you're teaching children! Feel free to ask any questions here.

**Reply by martiansparlor** [reply]
Thanks ! It totally makes senses now.

I'm planning to do a beginer's documentation for making a platformer in Kaboom.
You can see here a first draft I did with the 2000 version which has many things to imporve on : http://martiansparlor.com/ateliers/kaboom/

It really is a wonderful tool, thanks for your work and dedication !
